funnel/views/auth_client.py

```diff
@@ -88,317 +88,355 @@ def client_new():
     )
 
 
-@app.route('/apps/<key>')
-@load_model(AuthClient, {'buid': 'key'}, 'auth_client', permission='view')
-def client_info(auth_client):
-    if auth_client.user:
-        permassignments = AuthClientUserPermissions.all_forclient(auth_client).all()
-    else:
-        permassignments = AuthClientTeamPermissions.all_forclient(auth_client).all()
-    return render_template(
-        'auth_client.html.jinja2',
-        auth_client=auth_client,
-        permassignments=permassignments,
-    )
+@route('/account/apps/<key>')
```

Comment:
By convention, this should be `<client>`. However, since the vocabulary is moving (step at a time) from "client" to "auth client" to "client app", maybe `<app>`